recipes/genomekit/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: genomekit
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepgenomics/GenomeKit/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8793fb15cba3f572879996d47d6320c8ec983995c6f713848f53ca5dae7e5844
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

Comment:
I'm a bit confused to see Tensorflow as a reason here and not seeing it in the dependencies. Especially as the comment is about wheels which is independent of conda-forge.

Reply:
### Context:

This is a newly open sourced project, which is being used commonly on TPUs, while also loading tensorflow separately (it doesn't really make sense to make tensorflow a dependency of this package).

In order to run Tensorflow on TPUs, you need libtpu, which is not possible to compile from source, and until recently only offered a wheel for Py3.8.
-----------------

We've been avoiding offering support for py>3.8 due to C Extention API changes, but a change is in the works and will be made available soon.
I'll change the comment to reflect that and remove the TPU reference.